### ISSUE TITLE
Main menu list should not contain non <li> elements 

### ISSUE THREAD

**Issue by thanyabegum** [pulibrary/dpul#1216]
The menu that displays when a user clicks on their name is made of a list that contains non ```<li>``` elements. In this case, that element is a button. More details below.

<img width="533" alt="Screen Shot 2021-11-05 at 2 23 53 PM" src="https://user-images.githubusercontent.com/66143640/140560459-9c070276-b03c-47d2-be3f-92f34f883099.png">

[Learn more about this failing Lighthouse test.
](https://web.dev/list/?utm_source=lighthouse&utm_medium=devtools)

**Comment by thanyabegum** [comment]
The main menu component comes from the Lux design system, so the changes needed to solve this issue would need to be made in that codebase, specifically in the [MenuBar component file](https://github.com/pulibrary/lux/blob/008a20c9b36755e52b2ac158d08458a3739c1013/src/patterns/MenuBar.vue#L85). One solution would be to wrap the button in an ```<li>``` tag and see if that removes the accessibility error.
